stl/inc/chrono

```diff
@@ -58,20 +58,19 @@ namespace chrono {
     };
 
 #if _HAS_CXX20
+    template <class _Clock, class = void>
+    inline constexpr bool _Is_clock_v = false;

```

Comment:
Couldnt we just name this `is_clock_v` and be done with it=

Reply:
The additional `class = void` would definitely be observable through IntelliSense and possibly observable to conformant (but weird) code; we have permission to add extra arguments to functions but I don't know if it extends to variable templates. Generally we've refrained from modifying the Standard type traits in this way.

Reply:
Thanks a lot for the explanation. I was wondering about something in that direction but thought asking is better ;)